docs/docs/APIdocs/layers/nn/Convolution Layers/SpatialShareConvolution.md

```diff
@@ -0,0 +1,110 @@
+## SpatialShareConvolution ##
+
+**Scala:**
+```scala
+val layer = SpatialShareConvolution[Double](nInputPlane, nOutputPlane, kW, kH, dW, dH,
+      padW, padH)
+```
+**Python:**
+```python
+layer = SpatialShareConvolution(nInputPlane, nOutputPlane, kW, kH, dW, dH, padW, padH)
+```
+
+ Applies a 2D convolution over an input image composed of several input planes.
+ The input tensor in forward(input) is expected to be
+ a 3D tensor (nInputPlane x height x width).
+
+ This layer has been optimized to save memory. If using this layer to construct multiple convolution
+ layers, please add sharing script for the fInput and fGradInput. Please refer to the ResNet example.
+
+**Scala example:**
```

Comment:
The API of SpatialShareConvolution and SpatialConvolution are the same. SpatialShareConvolution itself has already compressed its fInput and fGradInput.